pkg/i2gw/providers/istio/converter.go

```diff
@@ -600,7 +611,7 @@ func (c *converter) convertVsHTTPRoutes(virtualService metav1.ObjectMeta, istioH
 				OwnerReferences: virtualService.OwnerReferences,
 				Finalizers:      virtualService.Finalizers,
 			},
-			hostnames:   hostnames,
+			hostnames:   allowedHostnames,
```

Comment:
```suggestion
			hostnames:   c.convertHostnames(istioHTTPHosts),
```

Reply:
is it good to call that function in a loop?